hello_leds/README.md

```diff
@@ -1,8 +1,36 @@
-# `hello_leds`
+# Hello LEDs
 
-Functionaly similar to `blinky`, using the Linux leds class instead of manually asserting the GPIO line.
+Functionally similar to `blinky`, but showing how the Linux `/sys/class/leds` interface can be used to blink the LED(s) instead of manually turning them on and off.
```

Comment:
Suggest adding this link to the sys class reference: http://elixir.free-electrons.com/source/Documentation/leds/leds-class.txt?v=4.10

P.S. check out that the name of the Linux browsing tool is "elixir".